htdocs/admin/invoice.php

```diff
@@ -826,6 +826,14 @@
 print ajax_constantonoff('INVOICE_CHECK_POSTERIOR_DATE');
 print '</td></tr>';
 
+
+// Disallow to classify billed an order without invoice
+print '<tr class="oddeven"><td>'.$langs->trans("InvoiceClassifyBilledSupplierOrderWithoutInvoice"). '&nbsp;' ;
+print $form->textwithpicto('', $langs->trans("InvoiceClassifyBilledSupplierOrderWithoutInvoiceHelp"), 1, 'help') . '</td>';
+print '<td class="left" colspan="2">';
+print ajax_constantonoff('WORKFLOW_DISABLE_CLASSIFY_BILLED_FROM_ORDER');
```

Comment:
Constant with name WORKFLOW_* are used when an action into a module generate another into another module (and provided by module Workflow).
Here we want to disable a feature in module order that disable ability to change status billed with no interaction in other module. So option should not be named WORKFLOW_*
So can you rename the option into
ORDER_DISABLE_CLASSIFY_BILLED_FROM_ORDER